packages/angular/src/tracing.ts

```diff
@@ -143,10 +143,13 @@ export class TraceService implements OnDestroy {
         (event.state as unknown as RouterState & { root: ActivatedRouteSnapshot }).root,
       );
 
+      if (route) {
+        getCurrentScope().setTransactionName(route);
+      }
```

Comment:
Note, I'm deliberately not guarding this by the root span b/c we also want to update the scope's `transactionName` regardless of the presence or state of a (root) span.